site/content/docs/user/known-issues.md

```diff
@@ -32,7 +32,7 @@ description: |-
 * [Pod Errors Due to "too many open files"](#pod-errors-due-to-too-many-open-files) (likely [inotify] limits which are not namespaced)
 * [Docker Permission Denied](#docker-permission-denied) (ensure you have permission to use docker)
 * [Windows Containers](#windows-containers) (unsupported / infeasible)
-* [Non-AMD64 Architectures](#non-amd64-architectures) (images not pre-built yet)
+* [Non-AMD64 Architectures](#nonamd64-architectures) (images not pre-built yet)
```

Comment:
the anchor is indeed  nonamd64...`, but unclear to me why that is (i.e. what is generating it without the `-`)

```
<h2 id="nonamd64-architectures">Non-AMD64 Architectures<a href="#nonamd64-architectures" class="hanchor" arialabel="Anchor"> 🔗︎</a> </h2>
```

https://kind.sigs.k8s.io/docs/user/known-issues/#non-amd64-architectures

/lgtm
/ok-to-test

Reply:
hugo generates the element IDs